content/300-guides/200-deployment/100-deployment.mdx

```diff
@@ -100,7 +105,28 @@ _Transaction mode_ offers a connection for every transaction – a requirement f
 
 </details>
 
-## Serverless (FaaS)
+## Deploying database changes with Prisma Migrate
+
+To apply pending migrations to development, staging, or testing environments, run the `migrate deploy` command as part of your CI/CD pipeline:
+
+```terminal
+prisma migrate deploy
+```
+
+Exactly when to run `prisma migrate deploy` depends on your platform. For example, a simplified [Heroku](deploying-to-heroku) workflow includes:
+
+1. Ensuring the `./prisma/migration` folder is in source control
+2. Running `prisma migrate deploy` during the [release phase](https://devcenter.heroku.com/articles/release-phase)
+
+> **Note**: `migrate deploy` should generally be part of an automated CI/CD pipeline, and we do not recommend running this command locally to deploy changes to a production database (for example, by temporarily changing the `DATABASE_URL` environment variable).
+
```

Comment:
Beware that in order to run the `prisma migrate deploy` command, you need access the `prisma` dependency that is typically added to the `devDependencies`. Some platforms like Vercel, prune development dependencies during the build, thereby preventing you from calling the command. This can be worked around by making the `prisma` a production dependency, by moving it to `dependencies` in your `package.json`.